expression-language/src/test/java/io/zeebe/el/impl/StaticExpressionTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.el.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.zeebe.el.ResultType;
+import org.junit.Test;
+
+public class StaticExpressionTest {
```

Comment:
This test could be merged with `EvaluationResultTest`. Instead of test the concrete implementation `StaticExpression`, it would test the `ExpressionLanguage` / `EvaluationResult` API.

IMO: I would prefer an API test over the `StaticExpression` test because the logic in `StaticExpression` is not complicated and it does not guarantee that the API behaves the same (e.g. if the class is used differently or not used at all).

Reply:
agreed and implemented